CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Fixed issue with pyyaml version for Python 3.4 by requiring pyyaml version <=5.2
 
 ### Changed
+- Updated cloudpickle and joblib dependencies. (#349)
+- CivisML uses platform aliases instead of hard-coded template IDs. (#341)
```

Comment:
This entry duplicates the next line just below.

Reply:
merge conflicts FTW